cmd/antrea-agent/agent.go

```diff
@@ -320,6 +313,51 @@ func run(o *Options) error {
 		return err
 	}
 
+	// Initialize denyConnStore, conntrackConnStore, flowExporter.
+	var denyConnStore *connections.DenyConnectionStore
+	var conntrackConnStore *connections.ConntrackConnectionStore
+	var flowExporter *exporter.FlowExporter
+	if features.DefaultFeatureGate.Enabled(features.FlowExporter) {
+		v4Enabled := config.IsIPv4Enabled(nodeConfig, networkConfig.TrafficEncapMode)
+		v6Enabled := config.IsIPv6Enabled(nodeConfig, networkConfig.TrafficEncapMode)
+
+		denyConnStore = connections.NewDenyConnectionStore(
+			ifaceStore,
+			proxier,
+			o.activeFlowTimeout,
+			o.idleFlowTimeout,
+			o.staleConnectionTimeout)
+		networkPolicyController.SetDenyConnStore(denyConnStore)
```

Comment:
Thanks Antonin and Srikar. In the latest commit, I have 
1. used setter for setting `denyConnStore` of `networkPolicyController`
2. separate connection stores & flowExporter initialization and go routines creation

To make it more consistent, shall I move the initialization of `NPLAgent`, `agentQuerier`, `apiServer` above the creation of all go routines?